index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio by Abenezer Samuel</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="toolbar">
+    <nav class="nav-items">
+      <h2 class="logo">Ebenezer</h2>
+      <img src="./image/menu.png" alt="menu bar image" />
```

Comment:
- The image here is broken, let's kindly fix this as seen in the image below:

![image](https://github.com/abenisamuel11/mobile-set-up-season2/assets/59397159/0098baaf-faf3-4f5c-a7b7-ef83882ec3c3)